math/svmlin/Portfile

```diff
@@ -0,0 +1,28 @@
+# -*- coding: utf-8; mode: tcl; tab-width: 4; indent-tabs-mode: nil; c-basic-offset: 4 -*- vim:fenc=utf-8:ft=tcl:et:sw=4:ts=4:sts=4
+
+PortSystem          1.0
+
+name                svmlin
+version             1.0
+categories          math
+license             GPL-2
+maintainers         {@barracuda156 gmail.com:vital.had} openmaintainer
+description         Fast linear SVM solvers for supervised and semi-supervised learning
+long_description    {*}${description}
+homepage            https://vikas.sindhwani.org
+master_sites        ${homepage}/
+distname            ${name}-v${version}
+checksums           rmd160  730132b7c06e67436222abf88935570d9e425e7f \
+                    sha256  2647a23805dd6e6b35bbad47e08c1202446fbd8c8045256993a41b902a2b37e0 \
+                    size    1068173
+
+post-patch {
+    reinplace "s|g++|${configure.cxx}|" ${worksrcpath}/Makefile
+    reinplace "s|-Wall -O3|-Wall -O3 [get_canonical_archflags cxx]|" ${worksrcpath}/Makefile
+}
+
+use_configure       no
```

Comment:
In most cases that you have software that doesn't use `configure`, you can probably make use of the `makefile` portgroup:

```
PortSystem          1.0
PortGroup           makefile
```

The `makefile` portgroup ensures all the right compiler settings for `make`, and invokes `use_configure no` by default.

Reply:
@herbygillot Did not know about that, thank you. Will try it.